src/argilla/server/alembic/versions/26721800a9f3_add_last_activity_at_to_datasets_table.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""add last_activity_at to datasets table
+
+Revision ID: 26721800a9f3
+Revises: 1e629a913727
+Create Date: 2023-10-18 16:39:03.977941
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "26721800a9f3"
+down_revision = "1e629a913727"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("datasets", sa.Column("last_activity_at", sa.DateTime(), nullable=False))
```

Comment:
for datasets that were created before this migration is applied, we have to give a default value if `nullable=False`, right?